src/mixins/_mixins.scss

```diff
@@ -342,3 +342,23 @@
   color: $color;
   font-size: $fd-size;
 }
+
+@mixin fd-icon-overwrite {
+  &[class*=sap-icon] {
```

Comment:
I've noticed this before in Icon implementation, and I think we should switch `[attribute*=value]` selector to `[attribute^=value]` because of performance. But we can leave this for later talk.

Reply:
That's not in the scope of the PR